interface/main/tabs/main.php

```diff
@@ -331,14 +335,24 @@ function assignPatientDocuments(patientId) {
     // prepare Issues popup link global that is used in creating the menu
     $GLOBALS['allow_issue_menu_link'] = ((AclMain::aclCheckCore('encounters', 'notes', '', 'write') || AclMain::aclCheckCore('encounters', 'notes_a', '', 'write')) &&
         AclMain::aclCheckCore('patients', 'med', '', 'write'));
-    ?>
 
-    <?php require_once("templates/tabs_template.php"); ?>
-    <?php require_once("templates/menu_template.php"); ?>
+    // we use twig templates here so modules can customize some of these files
+    // at some point we will twigify all of main.php so we can extend it.
+    echo $twig->render("interface/main/tabs/tabs_template.html.twig", []);
+    echo $twig->render("interface/main/tabs/menu_template.html.twig", []);
+    // TODO: patient_data_template.php is a more extensive refactor that could be done in a future feature request but to not jeopardize 7.0.3 release we will hold off.
+    ?>
     <?php require_once("templates/patient_data_template.php"); ?>
-    <?php require_once("templates/therapy_group_template.php"); ?>
-    <?php require_once("templates/user_data_template.php"); ?>
-    <?php require_once("menu/menu_json.php"); ?>
```

Comment:
can menu/menu_json.php script be removed?

Reply:
Maybe i am misunderstanding this. But isn't menu/menu_json.html.twig basically replacing menu/menu_json.php ? (that's why I thought menu/menu_json.php could be removed)

Reply:
Are you meaning remove the file from the filesystem?  I thought your comment was in reference to the line of code.  Yes it should be removed, thought I had removed it but can see in the commit its still there.  Will remove it from the filesystem.

Reply:
Removed.